src/releasers/node.ts

```diff
@@ -101,6 +110,15 @@ export class Node extends ReleasePR {
       })
     );
 
+    updates.push(
+      new SamplesPackageJson({
+        path: 'samples/package-lock.json',
```

Comment:
The `samples/package.json` convention is supporting Cloud's own convention, where we ship [samples with their own `package.json`](https://github.com/googleapis/nodejs-secret-manager/blob/master/samples/package.json).

I don't expect too many other folks in the community have this nested `package.json`, so I'm thinking we could probably drop lines `113 - 121`, and just add support for the top level `package-lock.json`.

Reply:
Done